docs/framework/package-generator/javascript-package.md

```diff
@@ -214,11 +214,11 @@ To change the ESLint configuration, edit the [.eslintrc.js](https://github.com/c
 
 ### Why are the predefined ESLint rules recommended?
 
-To make CKEditor 5 plugins compatible with each other, we needed to introduce certain limitations when importing files from packages. To learn more, visit the {@link installation/advanced/dll-builds DLL guide} and {@link framework/guides/contributing/code-style#dll-builds-ckeditor5-rulesckeditor-imports see a detailed explanation} about the limitations.
+To make CKEditor 5 plugins compatible with each other, we needed to introduce certain limitations when importing files from packages. To learn more, visit the {@link installation/advanced/dll-builds DLL guide} and {@link framework/contributing/code-style#dll-builds-ckeditor5-rulesckeditor-imports see a detailed explanation} about the limitations.
 
 ## Translations
 
-Packages created by this tool, just like the entirety of the CKEditor 5 ecosystem, include full support for localization. If you wish to include translations for your package, visit the {@link framework/guides/deep-dive/localization dedicated translation guide} and learn more.
+Packages created by this tool, just like the entirety of the CKEditor 5 ecosystem, include full support for localization. If you wish to include translations for your package, visit the {@link framework/deep-dive/localization dedicated translation guide} and learn more.
```

Comment:
This link doesn't work, probably missing a redirect. It should probably point here: `framework/deep-dive/ui/localization.html`.